frontend/src/views/c-authorship.vue

```diff
@@ -627,7 +619,7 @@ export default {
 
       res.sort((a, b) => b.lineCount - a.lineCount).forEach((file) => {
         // hide files over total char count limit
-        if (!file.isIgnored && !file.isBinary && file.active) {
+        if (!file.isIgnored && !file.isBinary && file.active && file.charCount) {
```

Comment:
May I check why is `file.charCount` added to the condition check? If `file.charCount` is 0, it will not enter the if branch. This is a little different from the original logic.

Reply:
The check was added because in the following line we check `file.charCount <= SINGLE_FILE_CHAR_COUNT_THRESHOLD`, but `charCount` is an optional field.
Thanks for pointing this out, here I think we have to do an explicit `file.charCount !== undefined` check to still allow the code to enter the branch if `file.charCount` is 0 like you mentioned.

If we check `file.charCount !== undefined` it technically won't be changing any logic, since [charCount is only set when `!file.isIgnored && !file.isBinary`.](https://github.com/reposense/RepoSense/blob/897c5fb116ea9ff51d6bd2275238ce5b966547ec/frontend/src/views/c-authorship.vue#L603), so it will always be true.